### DISCUSSION TITLE
UI options for adding/managing indexed columns

### DISCUSSION THREAD

**Discussion by ChrisThackrey** [supabase/supabase#3003] [Feature Requests]
Enjoying my experience with supabase so far! 

Working with indexes, text-search a lot in my current project and wondering if there are plans to extend Postgres operations like this to the UI in future versions?

It would be nice to visualize which columns have indexes and manage indexing via UI options. There is a whole section of the docs dedicated to full-text-search in the SQL editor, but it seems like a common operation that could be automated by adding/removing/listing/updating index types directly in the UI.

**Comment by saltcod** [comment]
Hey there! We've just added a new UI for this, available here:
https://supabase.com/dashboard/project/_/database/indexes

Please let us know if you've got any feedback!
